doc/changes/latest.inc

```diff
@@ -39,6 +39,8 @@ Enhancements
 ~~~~~~~~~~~~
 .. - Add something cool (:gh:`9192` **by new contributor** |New Contributor|_)
 
+- Add ability to export EDF+ and BDF files using `mne.export.export_raw` (:gh:`9643` by `Adam Li`_)
```

Comment:
```suggestion
- Add ability to export EDF+ files using `mne.export.export_raw` (:gh:`9643` by `Adam Li`_)
```

Reply:
Good catch. I switched this back and forth like 3 times, so I forgot on the last one :p.